File: Conditionals/time_day.c
#include <stdio.h>
#include <time.h>

int hour;

int main(void){
    time_t now;
    struct tm*now_tm;

    now = time(NULL);
    now_tm = localtime(&now);
    hour = now_tm->tm_hour;
    printf("%d\n", hour);

    if (hour <= 12){
        printf("mroning... -n-\n");
    }else if (hour <= 18){
        printf("Good afternoon OwO\n");
    }else if (hour <= 20){
        printf("GoOd EvEnInG YAA- >:)\n");
    }else{
        printf("GOOD EVENING. GO TO SLEEP o^O\n");
    }

    return 0;
}

File: Conditionals/time_day.py
import datetime

now = datetime.datetime.now().hour

if now <= 12:
    print("mroning... -n-")
elif now <= 18:
    print("Good afternoon OwO")
elif now <= 20:
    print("GoOd eVeNiNg ovo")
else:
    print("GOOD MOrNiNg YAA- >w<")

print(now)